woocommerce/woocommerce-gutenberg-products-block
# Remove .gitattributes file

Fixes #4725

At the moment, the repo zip file does not contain files that are needed for development. Download this file, does not enable anyone to develop on top of it. The development is only possible when fetching this repo via git. This defeats the purpose of the repo zip file.

In https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/trunk/.gitattributes, we defined which files and folders to exclude from the deploy zip when running `npm run package-plugin:deploy`. However, these rules also exclude these files and folders from the repo zip file, which is not intended. In addition, we also use https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/trunk/bin/copy-plugin-files.sh to exclude files and folders from the deploy zip file. 

I removed the `.gitattributes` file completely and when running `npm run package-plugin:deploy`, the deployment build does not change. This means that `.gitattributes` has no more influence on the deploy zip.

### Testing

1. Head over to https://github.com/woocommerce/woocommerce-gutenberg-products-block/tree/fix/4725-add-build-files-to-repo-zip.
2. Download the repo zip file.
3. Unzip the file locally.
4. Ensure that the zip file contains all files that are needed for development.
5. Open this folder in the terminal.
6. Run `composer i && npm i && npm run build` to verify that development is possible.
